system/lib/libc/musl/src/internal/pthread_impl.h

```diff
@@ -22,7 +22,6 @@ struct pthread {
 	// by direct pointer arithmetic in worker.js.
 	int threadStatus; // 0: thread not exited, 1: exited.
 	int threadExitCode; // Thread exit code.
-	int tempDoublePtr[3]; // Temporary memory area for double operations in runtime.
```

Comment:
Was this simply not used?

Reply:
Yeah, looks like I had intended that to live in pthread area at some point, but never got around to doing that. I think the stack area is a better fit, since pthread struct is musl code and pthreads specific, so fewer changes there.